compiler.go

```diff
@@ -956,7 +999,7 @@ func (c *Compiler) checkCyclicImports(
 func (c *Compiler) compileModule(
 	node parser.Node,
 	moduleName, modulePath string,
-	src []byte,
+	src []byte, isFile bool,
```

Comment:
can you put `isFile bool,` in a separate line?

Reply:
Ok